LondonBritishLibrary/orient/BLorient616.xml

```diff
@@ -149,7 +149,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <msItem xml:id="ms_i3.4">
 <locus from="152r"/>
 <title xml:lang="en" type="complete">Another homily of John Chrysostom on the same subject</title>
-<incipit xml:lang="en">ድርሳን፡ ዘቅዱስ፡ ገብርኤል፡ በእንተ፡ ዘዜነዋ፡ ገብርኤል፡ ለማርያም፡</incipit>
+<incipit xml:lang="gez">ድርሳን፡ ዘቅዱስ፡ ገብርኤል፡ በእንተ፡ ዘዜነዋ፡ ገብርኤል፡ ለማርያም፡</incipit>
```

Comment:
is the citation correct? should it not be ድርሳን፡ ዘቅዱስ፡ ዮሓንስ፡ ...

Reply:
Like this in the catalogue, I added a clarification, thanks for noticing!